src/librustdoc/html/format.rs

```diff
@@ -412,6 +412,16 @@ impl clean::GenericBound {
                 })?;
                 ty.print(cx).fmt(f)
             }
+            clean::GenericBound::Use(args) => {
+                write!(f, "use<")?;
```

Comment:
`<` only if `f.alternate()` otherwise `&lt;`

Reply:
Ah, that's quite important. Thanks for the heads up -- I never really understood what the alternate formatting was for in rustdoc.